### PR TITLE
Add tcp port name to service

### LINKED ISSUE #527: Add e2e for SRV record support
**What would you like to be added**:
Add e2e for SRV record support

**Why is this needed**:
To test the changes added as a part of SRV record

### PR DESCRIPTION
**Pull request by aswinsuryan**
This is required to test SRV records

Fixes: submariner-io/lighthouse#527

Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

File: test/e2e/framework/services.go [hunk 1]
```diff
@@ -66,7 +67,8 @@ func (f *Framework) CreateTCPService(cluster ClusterIndex, selectorName string,
 }
 
 func (f *Framework) NewNginxService(cluster ClusterIndex) *corev1.Service {
-	var port int32 = 80
+	var tcpPort int32 = 80
+        var metricsPort int32 = 8183
```
Comment (dfarrell07) [inline test/e2e/framework/services.go]: Is this indentation wrong? It looks wrong, but I'm surprised no linter is failing.

Comment (aswinsuryan) [reply test/e2e/framework/services.go]: fixed , it was an issue with "space" vs "tab"

**Review by skitt — APPROVED** [review]

File: test/e2e/framework/services.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ func (f *Framework) CreateTCPService(cluster ClusterIndex, selectorName string,
 		Spec: corev1.ServiceSpec{
 			Ports: []corev1.ServicePort{{
 				Port:       int32(port),
+                                Name:       "tcp",
```
Comment (tpantelis) [inline test/e2e/framework/services.go]: Indentation looks off.

File: test/e2e/framework/services.go [hunk 3]
```diff
@@ -78,13 +80,24 @@ func (f *Framework) NewNginxService(cluster ClusterIndex) *corev1.Service {
 			Type: "ClusterIP",
 			Ports: []corev1.ServicePort{
 				{
-					Port:     port,
+					Port:     tcpPort,
+                                        Name:     "http",
```
Comment (tpantelis) [inline test/e2e/framework/services.go]: Indentation looks off.

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**